Content.Server/Shuttles/Systems/ShuttleSystem.FasterThanLight.cs

```diff
@@ -478,12 +479,28 @@ private void UpdateFTLArriving(Entity<FTLComponent, ShuttleComponent> entity)
 
         if (!Exists(entity.Comp1.TargetCoordinates.EntityId))
         {
```

Comment:
Honestly, this should just return and log an error if the target map doesn't exist, because that is a programming error. Could also make it just give control back to the shuttle if it fails, so that they can FTL back into a proper map.